lib/dynamoid/undumping.rb

```diff
@@ -284,7 +284,7 @@ def process(value)
 
     class BinaryUndumper < Base
       def process(value)
-        Base64.strict_decode64(value)
+        value.read
```

Comment:
It seems `StringIO#read` makes a copy of a string (`#read` accepts a length parameter so in general case it can return a substring what is left to "read"), so `StringIO#string` looks slightly better from the performance/memory footprint point of view.